src/gui.c

```diff
@@ -1559,13 +1591,22 @@ send_keypress_event (GdkEventKey *event)
 
     /* Get modifier state including this key press/release. */
     modifiers = get_modifier_mask ((event->type == GDK_KEY_PRESS) ? (state | mod) : (state & ~mod));
+    uzbl.gui_->current_key_modifiers = modifiers;
```

Comment:
this is opens for a segfault as modifiers is free()d at the end of this function. Saving the int state would be easier and just run get_modifier_mask when needed.